dts/bindings/sensor/nxp,mma8451q.yaml

```diff
@@ -0,0 +1,22 @@
+#
+# Copyright (c) 2018, Lars Knudsen
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+---
+title: MMA8451Q 3-axis Accelerometer
+id: nxp,mma8451q
+version: 0.1
+
+description: >
+    This is a representation of the MMA8451Q 3-axis Accelerometer
+    sensor
+
+inherits:
+    !include i2c-device.yaml
+
+properties:
+    compatible:
+      constraint: "nxp,mma8451q"
```

Comment:
Please add a node for this sensor to `boards/arm/frdm_kl25z/frdm_kl25z.dts`

Reply:
thanks for the tip - I *think* I got it right (at least data is flowing again ... and in m/s^2 ;)